docs/source/charge_isotope_deconvolution.rst

```diff
@@ -1,27 +1,27 @@
 Charge and Isotope Deconvolution 
 ================================
 
-A single mass spectrum contains measurements of one or more analytes and the
-m/z values recorded for these analytes. Most analytes produce multiple signals
-in the mass spectrometer, due to the natural abundance of carbon 13 (naturally
-occurring at ca. 1% frequency) and the large amount of carbon atoms in most
+A single :term:`mass spectrum` contains measurements of one or more analytes and the
+:term:`m/z` values recorded for these analytes. Most analytes produce multiple signals
+in the mass spectrometer, due to the natural abundance of carbon :math:`13` (naturally
+occurring at ca. :math:`1%` frequency) and the large amount of carbon atoms in most
 organic molecules, most analytes produce a so-called isotopic pattern with a
-monoisotopic peak (all carbon are C12) and a first isotopic peak (exactly one
-carbon atom is a C13), a second isotopic peak (exactly two atoms are C13) etc.
+monoisotopic :term:`peak`  (all carbon are :math:`C12`) and a first isotopic :term:`peak` (exactly one
```

Comment:
Please do not use math for chemical formulas. Chemical elements are *no mathematical variables*.
Use :sub: for stoichiometric coefficients and :sup: for isotopes.

Reply:
Or, even better, write your own :chem: role. https://github.com/OpenMS/OpenMS-docs/blob/develop/_ext/pathrole.py

Reply:
What about a `:chem:` alias for `:math:`?

Reply:
No the problem is that chemical formulas should never be typeset like mathematical formulas. Just run your code and you will see what I mean. Elements will look like variables for mathematical sets.

I would not mind the name.

Reply:
Ah I see what you mean, using \ce inside the math env is a good idea. A little bit wasteful since it creates pictures/Mathjax but it's ok.
Last things: 

- There are some formulas missing:
![Bildschirmfoto 2023-02-17 um 11 16 46](https://user-images.githubusercontent.com/8102638/219616870-5e21d5db-efd0-45ca-aae6-0838d519bbd5.png)

- Isotopes need to be typeset as :chem:`^{13}C`. But, be careful, whenever we are speaking about input to python as string, it still should be a raw string `(13)C`.

- M+H has problems:
![Bildschirmfoto 2023-02-17 um 11 19 46](https://user-images.githubusercontent.com/8102638/219617521-4c49dcb3-43cc-4728-96c1-a9c56bf66b2d.png)

Reply:
What do you mean the input should still be a raw string?

Reply:
@matteopilz If the docs say: "Create an EmpiricalFormula with the string `13C`" then of course you do not write it in chemical notation.